src/fetchingData.js

```diff
@@ -0,0 +1,201 @@
+//Pagination
+let pageNum = 1;
+let todayGMT = new Date();
+//Get yesterday's date
+todayGMT.setDate(todayGMT.getDate() - 1);
+let convertedToISO = todayGMT.toISOString();
+let url = `https://newsapi.org/v2/everything?q=trump&language=en&from=${convertedToISO}&sortby=popularity&apiKey=2b19d03fc1cf4cd7b62ac5bb15c98827&page=${pageNum}`;
+/*
+        Send the 20(default) articles through a series of
+        functions one at a time to populate our ul. 
+        "uri" should be undefined the first time, but not subsequent times
+        for purposes of pagination.
+*/
+function grabData(uri) {
```

Comment:
Rather than using `if/else` you can use default parameter values which will replace the parameter value when it is undefined. For example
`function grabData(uri = url){`
That way if `uri` is undefined, the `url` value will be applied instead